source/guides/postgres-migration.rst

```diff
@@ -0,0 +1,199 @@
+Migration guidelines from MySQL to PostgreSQL
+=============================================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+From Mattermost v8.0, PostgreSQL is our database of choice for Mattermost to enhance the platform’s performance and capabilities. Recognizing the importance of supporting the community members who are interested in migrating from a MySQL database, we have taken proactive measures to provide guidance and best practices. 
+
+To streamline the migration process and alleviate any potential challenges, we have prepared a comprehensive set of guidelines to facilitate a smooth transition. Additionally, we want to offer recommendations for various tools that have proven to be highly effective in simplifying your migration efforts.
+
+.. note::
+
+   These guidelines are in development and we are working to streamline the migration process. We plan to improve this guide by updating it as new information becomes available. It is essential to note that it does not encompass migration configurations for any plugins, such as Focalboard and Playbooks. If your system utilizes these plugins, we highly advise exercising patience until we incorporate the necessary configurations specifically tailored to ensure a smooth transition for those plugins as well. Please use this guide as a starting point and always backup your database before starting a migration.
+
+.. contents:: On this page:
+  :backlinks: top
+  :local:
+  :depth: 1
+
+Required tools
+--------------
+
+-  Install ``pgLoader``. See the official `installation
+   guide <https://pgloader.readthedocs.io/en/latest/install.html>`__.
+-  Install morph CLI via running the following command:
+
+   -  ``go install github.com/mattermost/morph/cmd/morph@v1``
+
+-  Optinally install ``dbcmp`` to compare the data after a migration:
+
+   -  ``go install github.com/mattermost/dbcmp/cmd/dbcmp@latest``
+
+Before the migration
+--------------------
+
+-  Backup your MySQL data.
+-  Confirm your Mattermost version. See the **About** modal for details. 
```

Comment:
@isacikgoz do we have a min MM version we would recommend? I would certainly like to call out being on a supported ESR.

Reply:
I think we can add that, for now it looks like the minimum support version is `v6.4` ESR as we require morph based schema migrations. So if they have version before `v6.4` they need to upgrade it first, then conduct the migration. Shall we add that info?

Reply:
@nab-77 added a note 👍